### ISSUE TITLE
Remove "jetpack" tags from existing add-ons and associated code

### ISSUE THREAD

**Issue by Ulf3000** [mozilla/addons-server#7189]
### Describe the problem and steps to reproduce it:

upload a sdk addon and amo always adds the keyword "jetpack" automatically, which already sucks ...

now i ported my first extension to webext , since it still has the @jetpack in its id, amo will still add jetpack to the keywords .

i suggest removing that feature completely also for older addons

thanx for listening :)

**Comment by diox** [comment]
@jvillalobos I also think this tag should be completely removed. Thoughts?

**Comment by jvillalobos** [comment]
We can remove it with all the legacy stuff later this year. We're not exposing tags on the new site now, so the current impact should be minimal.

**Comment by diox** [comment]
Note that the code to add the "jetpack" tag has already been removed a while ago in b1ab28d1b0ec441253c1cdcf235c50253b19f598 ; the idea here is to remove any extra code, but also add a migration to remove existing tags from add-ons in our database.

**Comment by AlexandraMoga** [comment]
I have tested the following scenarios:

-upgrading  a legacy add-on (one that uses a `@jetpack` guid) and making sure no tags were added after submission
- uploading a new addon using  a `@jetpack` guid and making sure no tag was automatically added

@diox has the migration to remove existing tags been run already? I did have a quick look at addons that potentially could have included a `jetpack` tag but the tag wasn't there. 
Also, let me know if there is anything more specific for us to test.
